ow_lander/src/ow_lander/frame_transformer.py

```diff
@@ -45,16 +45,16 @@ def transform_present(self, geometry, target_frame, source_frame, \
     stamped = None
     if isinstance(geometry, Point):
       stamped = PointStamped(
-        header=create_most_recent_header(source_frame), point=geometry)
+        header=create_header(source_frame, rospy.Time.now()), point=geometry)
```

Comment:
Shouldn't `timestamp` be used instead of `rospy.Time.now()`? In fact, `timestamp` isn't used at all in this method.